src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtAreaOfLawEndpointTest.java

```diff
@@ -0,0 +1,158 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.model.admin.AreaOfLaw;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.BEARER;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.objectMapper;
+
+@ExtendWith(SpringExtension.class)
+public class AdminCourtAreaOfLawEndpointTest extends AdminFunctionalTestBase {
+
+    private static final String ADMIN_COURTS_ENDPOINT = "/admin/courts/";
+    private static final String AREAS_OF_LAW_PATH = "courtAreasOfLaw";
+    private static final String AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG = "aylesbury-county-court-and-family-court/";
+    private static final String AYLESBURY_COURT_AREAS_OF_LAW_PATH = ADMIN_COURTS_ENDPOINT
+        + AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG + AREAS_OF_LAW_PATH;
+    private static final String AYLESBURY_COURT_AREAS_OF_LAW_NOT_FOUND_PATH = ADMIN_COURTS_ENDPOINT + "NotFound" + AREAS_OF_LAW_PATH;
+    private static final String TEST_AREA_OF_LAW_NAME = "Employment";
+    private static final int TEST_AREA_OF_LAW_ID = 34260;
+
+    /************************************************************* GET request tests section. ***************************************************************/
+    @Test
+    public void returnAreaOfLawForTheCourt() {
+        final var response = doGetRequest(
+            AYLESBURY_COURT_AREAS_OF_LAW_PATH,
+            Map.of(AUTHORIZATION, BEARER + authenticatedToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+
+        final List<AreaOfLaw> areasOfLaw = response.body().jsonPath().getList(".", AreaOfLaw.class);
+        assertThat(areasOfLaw).hasSizeGreaterThan(1);
+    }
+
+    @Test
+    public void shouldRequireATokenWhenGettingAreaOfLawForTheCourt() {
+        final var response = doGetRequest(AYLESBURY_COURT_AREAS_OF_LAW_PATH);
+        assertThat(response.statusCode()).isEqualTo(UNAUTHORIZED.value());
+    }
+
+    @Test
+    public void shouldBeForbiddenForGettingAreaOfLawForTheCourt() {
+        final var response = doGetRequest(
+            AYLESBURY_COURT_AREAS_OF_LAW_PATH,
+            Map.of(AUTHORIZATION, BEARER + forbiddenToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
+    }
+
+    @Test
+    public void shouldBeNotFoundForGetWhereAreaOfLawDoesNotExist() throws JsonProcessingException {
```

Comment:
@ashwini-mv I wonder what is the purpose of this test. The endpoint you are calling is /admin/courts/notFoundAreaOfLaw which does not exist at all. Of course this is going to return a not found. If you are specifically testing for court slug not found, the path should be /admin/courts/notFound/AreaOfLaw. If that is the case, it might be better to also test for the returned string body.

Reply:
This test itself is not needed, I have written test because that status code is mentioned. "/" is missing in the path. I will add that. But I don't think there is a need to test returned string body.